external_modules/src/peanut_moveit_interface/src/moveit_interface.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
+import rospy
+from util import *
+import numpy as np
+
+import moveit_commander
+import moveit_msgs.msg
+from geometry_msgs.msg import Pose
+
+from peanut_moveit_interface.srv import *
+
+
+class MoveitInterface(object):
+    """Interface between higher lvl code operating through botx and moveit
+    requires moveit to be launched and controllers to be up
+    """
+
+    def __init__(self):
+
+        # this variable is set when a service is being handled
+        self.processing_srv = False
+
+        # start the moveit commander
+        moveit_commander.roscpp_initialize(sys.argv)
+
+        # create robot commander instance
+        self.robot = moveit_commander.RobotCommander()
+
+        # initialize the planning scene
+        self.scene = moveit_commander.PlanningSceneInterface()
+
+        # initialize the arm move group
```

Comment:
This comment is not very descriptive, either change it or consider deleting.

Reply:
done